### ISSUE TITLE
User feedback: Sometimes the name of the card revealed when a dealt card is moved, is not announced

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#35]
From the user: "There are times when, upon moving a card from a dealt card pile to a target pile, Voiceover will not announce the next revealed card; for instance, it will say something like, "Moved 2 of hearts. Revealed in dealt card pile 6."

I've not been able to reproduce this. I've not found a code path where the name of the revealed card is not added to announcement, so perhaps the name is empty.

**Comment by gbarkerz** [comment]
This has been fixed by iOS Version 2.9, and can no longer be reproduced by the user who originally raised the issue.
